tools/sigclient/functionalQueries/sort_tail_eval(newField).json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
nit: dont use ( for the name of the file

Reply:
Addressed.